charts/osm/templates/osm-deployment.yaml

```diff
@@ -35,19 +37,13 @@ spec:
             "--webhook-config-name", "osm-webhook-{{.Values.OpenServiceMesh.meshName}}",
             "--ca-bundle-secret-name", "{{.Values.OpenServiceMesh.caBundleSecretName}}",
             "--certificate-manager", "{{.Values.OpenServiceMesh.certificateManager}}",
-            "--vault-host", "{{.Values.OpenServiceMesh.vault.host}}",
-            "--vault-protocol", "{{.Values.OpenServiceMesh.vault.protocol}}",
-            "--vault-token", "{{.Values.OpenServiceMesh.vault.token}}",
             "--cert-manager-issuer-name", "{{.Values.OpenServiceMesh.certmanager.issuerName}}",
```

Comment:
Do we need all the cert-manager flags here since we are using tresor?

Reply:
These flags provide support for Azure Key Vault, which will be added in the near future so I am adding the changes beforehand

Reply:
To ensure consistency with AKS, let's remove these for now and add them back when we add the support.

Reply:
I already have them in AKS too, so its already consistent. Let me know if you want me to remove them Arc.

Reply:
> These flags provide support for Azure Key Vault

If you already have them in AKS, then let's leave them. Just to confirm, you meant cert-manager, not Azure Key Vault right?

Reply:
yup that's right its cert-manager